decred/decred/util/helpers.py

```diff
@@ -189,7 +206,7 @@ def appDataDir(appName):
     # Fall back to standard HOME environment variable that works
     # for most POSIX OSes.
     if homeDir == "":
-        homeDir = os.getenv("HOME")
+        homeDir = os.getenv("HOME") or ""
```

Comment:
Mypy noticed that `os.getenv` could return `None`.

Reply:
```suggestion
        homeDir = os.getenv("HOME", "")
```

because

```
getenv(key, default=None)
    Get an environment variable, return None if it doesn't exist.
    The optional second argument can specify an alternate default.
    key, default and the result are str.
```

Reply:
Makes sense, thanks.